katello/hooks/post/30-upgrade.rb

```diff
@@ -1,13 +1,11 @@
 require 'fileutils'
 
 def db_seed
-  status = Kafo::Helpers.execute('foreman-rake db:seed')
-  fail_and_exit "Upgrade failed while seeding the DB" unless status
+  Kafo::Helpers.execute('foreman-rake db:seed')
```

Comment:
This drops the custom error message, which I think is OK, but would like to hear other peoples opinions.